app/models/page.rb

```diff
@@ -26,7 +26,7 @@ def self.total_visitors_for_phrase(phrase, start_date, end_date, sort_key: "tota
     date_range = start_date..end_date
 
     Page.joins(page_visits: [{ visit: [{ survey_visit: [{ survey: [{ survey_answers: [{ mentions: :phrase }] }] }] }] }])
-      .where("phrases.id" => phrase.id, "surveys.started_at" => date_range)
+      .where("surveys.started_at" => date_range)
```

Comment:
Aware this is only a draft at present, but why remove the constraint on phrase id?

Reply:
It was a whoopsie, back now!